src/main/core/Controller/APINew/User/UserController.php

```diff
@@ -247,6 +250,7 @@ public function enableUsersAction(Request $request): JsonResponse
         foreach ($users as $user) {
             if (!$user->isEnabled() && $this->checkPermission('ADMINISTRATE', $user)) {
                 $this->manager->enable($user);
+                $this->eventDispatcher->dispatch(SecurityEvents::USER_ENABLE, UserEnableEvent::class, [$user]);
```

Comment:
Should be dispatched from the `enable()` method. Not in the controller.

Reply:
Done, thx :)